shared/lib/token-utils.js

```diff
@@ -55,18 +64,18 @@ export const isTokenVerified = (tokenAddress, networkType) =>
     ? ETHER_TOKEN_VERIFIED_ADDRESSES.has(tokenAddress.toLowerCase())
     : true
 
-export const tokenDataFallback = (tokenAddress, fieldName, networkType) => {
+export const tokenDataOverride = (tokenAddress, fieldName, networkType) => {
   // The fallback list is without checksums
   const addressWithoutChecksum = tokenAddress.toLowerCase()
 
-  const fallbacksForNetwork = KNOWN_TOKENS_FALLBACK.get(networkType)
+  const overridesForNetwork = KNOWN_TOKENS_OVERRIDE.get(networkType)
   if (
-    fallbacksForNetwork == null ||
-    !fallbacksForNetwork.has(addressWithoutChecksum)
+    overridesForNetwork == null ||
```

Comment:
It might change the behavior in [unexpected ways](https://i.imgur.com/5pFXFbR.jpg), but I think we might want to change this to a `===` (threequals) while we're here.